sandbox/modules/identity/src/main/java/org/opensearch/identity/configuration/Initializable.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.identity.configuration;
+
+public interface Initializable {
```

Comment:
Why make this interface?

Reply:
Do you think it should be removed? A lot of this is modeled after the security plug-in with a lot of extraneous code removed out and core functionality kept in. This is only ever implemented once, I guess if there are multiple implementers then it would be more compelling to have an interface.

Reply:
I removed this interface which was only used once.